.github/actions/generate-brand-content/index.js

```diff
@@ -0,0 +1,100 @@
+const core = require('@actions/core');
+const fs = require('fs');
+const path = require('path');
+
+async function run() {
+  try {
+    const version = core.getInput('version', { required: true });
+    const commit = core.getInput('commit', { required: true });
+    const outputDir = core.getInput('output-dir', { required: true });
+    
+    // Get manifest from environment variable (set by validate-manifest action)
+    const manifestJson = process.env.BRANDING_MANIFEST;
```

Comment:
It is forbidden to use process.env.BRANDING_MANIFEST, all env vars must come from inputs. The only action that is responsible to manipulate the manifest json is the validate-manifest action